### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by ThomasDCW**

### CODE REVIEW

**Review by mastemm — DISMISSED** [review]

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Attention, if you use styled-components, all files for the style must be in styled-components

**Review by mastemm — DISMISSED** [review]

File: frontend/src/components/HomePage/style.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+import VS from "../../assets/homepage/VS.jpg";
+import HEROS from "../../assets/homepage/herosoftheday.jpg";
```
Comment (LoicBrassart) [inline frontend/src/components/HomePage/style.jsx]: You should not name your variables in CAPSLOCK but in camelCase

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please rename your variabls

**Review by mastemm — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
No more Evil ! =)

**Pull request merged**